torch/serialization.py

```diff
@@ -499,7 +499,16 @@ def save(
     torch._C._log_api_usage_once("torch.save")
     _check_dill_version(pickle_module)
     _check_save_filelike(f)
-
+    from torch._dynamo import OptimizedModule
+    if isinstance(obj, OptimizedModule):
```

Comment:
Please move this code to OptimizedModule `__getstate__` method!

Reply:
That was my initial intent but then I can't raise an error or else that'll break the minifier and a warning will be quite noisy again for the minifier

Reply:
Why is the minifier involved here? Doesn't the `__call__` the only method that has a custom behavior?